### PR TITLE
internal: copy testdata to routeguide

### PR DESCRIPTION
**Pull request by menghanl**
So it's not necessary to set the testdata file as flag.

Without this, using relative directory, the example only works if run in `example/routeguide`.

### CODE REVIEW

File: examples/route_guide/server/server.go [hunk 1]
```diff
@@ -45,13 +45,14 @@ import (
 	"github.com/golang/protobuf/proto"
 
 	pb "google.golang.org/grpc/examples/route_guide/routeguide"
+	"google.golang.org/grpc/examples/route_guide/testdata"
```
Comment (lyuxuan) [inline examples/route_guide/server/server.go]: why not just use the grpc/testdata package?

File: examples/route_guide/server/server.go [hunk 2]
```diff
@@ -233,6 +238,6 @@ func main() {
 		opts = []grpc.ServerOption{grpc.Creds(creds)}
 	}
 	grpcServer := grpc.NewServer(opts...)
-	pb.RegisterRouteGuideServer(grpcServer, newServer())
+	pb.RegisterRouteGuideServer(grpcServer, &server{})
```
Comment (lyuxuan) [inline examples/route_guide/server/server.go]: define `server`?

File: examples/route_guide/server/server.go (hunk 1 above)
Comment (menghanl) [reply examples/route_guide/server/server.go]: I wanted to keep example data separate from test data.

Also I renamed the package to `exampledata`.

File: examples/route_guide/server/server.go (hunk 2 above)
Comment (menghanl) [reply examples/route_guide/server/server.go]: Done......

**Pull request merged**